benchmark/test/test_script.py

```diff
@@ -41,12 +41,13 @@ def capture_stderr():
 ''' % _YAML_CONTROL_SERVICE_ADDRESS
 
 
-class ClusterConfigurationTests(SynchronousTestCase):
+class ClusterConfigurationTests(TestCase):
     """
     Tests for mapping cluster configuration to cluster.
     """
 
     def setUp(self):
+        super(ClusterConfigurationTests, self).setUp()
```

Comment:
My thinking was that synchronous is the default and isn't special in any way and thus doesn't need to be distinguished in any way.